linux_os/guide/services/ssh/ssh_server/sshd_do_not_permit_user_env/rule.yml

```diff
@@ -40,7 +40,7 @@ references:
     iso27001-2013: A.12.1.2,A.12.5.1,A.12.6.2,A.14.2.2,A.14.2.3,A.14.2.4
     cis-csc: 11,3,9
 
-ocil_clause: 'PermitUserEnvironment is not disabled'
+ocil_clause: 'PermitUserEnvironment is disabled'
```

Comment:
@Scronkfinkle This `ocil_clause` actually was correct. `ocil_clause` describes a situation when the rule should fail. For example, see rule https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/sssd/package_sssd_installed/rule.yml . That rule says that package sssd should be installed, but the ocil_clause there is `the package is not installed`.

Reply:
Gotcha, just pushed up a resolution!